src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,7 +76,10 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
-
+    public ArrayList<Person> mutableListView() {
```

Comment:
Since its a public, non-trivial method it should have header comments.
If not, developers who write client code for this class will not know how to use it.